### PR TITLE
Fix for scroll to subsection bug on docsify

### PR DESCRIPTION
**Pull request by beth-panx**
To test this, you will need to 
1. pull down the branch 
2. run `docsify serve docs`
3. check subsection link

### CODE REVIEW

**Comment by aarongustafson** [comment]
@beth-panx I don’t see a package.json. What do I need to install?

**Comment by beth-panx** [comment]
> @beth-panx I don’t see a package.json. What do I need to install?

The script is added to the index page. Nothing to install.

**Comment by aarongustafson** [comment]
> > @beth-panx I don’t see a package.json. What do I need to install?
> 
> The script is added to the index page. Nothing to install.

I meant to run docsify :-)

File: docs/index.html [hunk 1]
```diff
@@ -65,6 +65,29 @@
   <script src="//cdn.jsdelivr.net/npm/docsify@4.12.2"></script>
   -->
 
+  <!-- Hacky fix for scroll to subsection bug on docsify https://github.com/docsifyjs/docsify/issues/351-->
+  <script>
+    (function () {
+    function create() {
+        return (hook) => {
```
Comment (aarongustafson) [inline docs/index.html:72]: To enable better backward compatibility, you might consider switching from a fat arrow to an actual function definition. Also swapping `const` and `let` for `var`. If you don’t care about older browsers, it’s fine.

**Review by aarongustafson — APPROVED** [review]

**Pull request merged**